src/components/ViewQuiz.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+
+import { Card, ListGroup } from 'react-bootstrap';
+import { connect } from 'react-redux';
+
+import ViewQuestions from './ViewQuestions';
+
+class ViewQuiz extends Component {
+  constructor(props) {
+    super(props);
+  }
+  render() {
+    const { id } = this.props.match.params;
+    const { quizzes, questions } = this.props;
+    const quiz = quizzes.find(quiz => quiz.id == id);
```

Comment:
Pls use `===` and `!==` over `==` and `!=`

Reply:
fixed